src/resources/views/components/header.blade.php

```diff
@@ -0,0 +1,11 @@
+<header>
+    <div class="row">
+        <div class="col" style="width: 10%;max-width: 15%;"><img src="img/header/summa.png" style="width: 100%;"></div>
+        <div class="col" style="padding-right: 40px;">
+            <div class="row" style="height: 80%;">
+                <div class="col d-flex d-sm-flex d-md-flex justify-content-end align-items-center justify-content-sm-end align-items-sm-center justify-content-md-end align-items-md-center" style="height: 70%;min-height: 90%;margin-top: 0px;"><img class="d-lg-flex d-xl-flex" src="img/header/AMI_LOGO.png" style="width: 90%;"></div>
```

Comment:
Use SASS instead of inline style, and use `asset('img/header/AMI_LOGO.png')` for image source. Also use lower case letters wherever possible